dubbo-xds/src/main/java/org/apache/dubbo/registry/xds/util/protocol/AbstractProtocol.java

```diff
@@ -248,13 +196,30 @@ private void returnResult(T result) {
 
         @Override
         public void onCompleted() {
-            logger.info("xDS Client completed, requestId: " + requestId);
-            clear();
-        }
-
-        private void clear() {
-            requestObserverMap.remove(requestId);
+            logger.info("xDS Client completed");
         }
     }
 
+    private void triggerReConnectTask() {
+        AtomicBoolean isConnectFail = new AtomicBoolean(false);
+        ScheduledExecutorService scheduledFuture = ApplicationModel.defaultModel().getFrameworkModel().getBeanFactory()
+            .getBean(FrameworkExecutorRepository.class).getSharedScheduledExecutor();
```

Comment:
Should fetch `ApplicationModel` in contructor. `ApplicationModel.defaultModel()` is forbidden here.